style/modules/pressModule.css

```diff
@@ -0,0 +1,37 @@
+.pressModule {
+    display: flex;
+    /* overflow: hidden; */
+    background: #f5f5f5;
+    /* width: 1080px; */
+    height: 250px;
```

Comment:
I think we'll want this to be a min-height. you can verify it by testing with some longer text:

https://www.lipsum.com/

Reply:
<img width="1257" alt="Screen Shot 2020-10-14 at 9 34 30 PM" src="https://user-images.githubusercontent.com/733916/96077688-524a0300-0e65-11eb-805a-3cf9f99b205b.png">